masonry/src/contexts.rs

```diff
@@ -540,11 +488,34 @@ impl_context_method!(WidgetCtx<'_>, EventCtx<'_>, LifeCycleCtx<'_>, {
 // --- MARK: OTHER METHODS ---
 // Methods on all context types except PaintCtx and AccessCtx
 impl_context_method!(
-    WidgetCtx<'_>,
+    MutateCtx<'_>,
```

Comment:
Are these meaningful on `MutateCtx`?

Reply:
Maybe.

Either way, I'm trying to consider these methods less in terms of "does this context absolutely need it" and more "does it break something if this context has it". I think the context APIs are easier to reason about that way.

Reply:
It makes reuse of the `mutate_callbacks` vector potentially fraught, and the semantics aren't especially clear.

Effectively, if you're calling this, you should be using a different method on the `MutateCtx` which you already have...

Reply:
Agree, I don't see benefits of this yet (other than being confusing, or allure users to "hacky" workarounds). I think we should see how far we get without it on `MutateCtx`. (I'm specifically also talking about `mutate_callbacks` not sure about the rest, that may be useful)

Reply:
Will keep as-is for now, with a TODO comment. We can reconsider when passes are all implemented.